float64 posicion_x_robot
float64 posicion_y_robot
float64 robot_a
float32[] centroides_x
float32[] centroides_y
#int8[] mapa_costos
#uint32 width
#uint32 height
#float32 resolution
-----------------

#float32[] centroides_x
#loat32[] centroides_y
float64 obj_x
float64 obj_y
